codegen/src/main/ops/org/nd4j/codegen/ops/Image.kt

```diff
@@ -0,0 +1,272 @@
+package org.nd4j.codegen.ops
+
+import org.nd4j.codegen.api.AtLeast
+import org.nd4j.codegen.api.Language
+import org.nd4j.codegen.api.doc.DocScope
+import org.nd4j.codegen.dsl.*
+import org.nd4j.codegen.api.DataType.*
+
+
+
+fun SDImage() =  Namespace("SDImage"){
+    val namespaceJavaPackage = "org.nd4j.linalg.api.ops.custom"
+    Op("CropAndResize") {
+        javaPackage = "org.nd4j.linalg.api.ops.impl.image"
+        javaOpClass = "CropAndResize"
+        Input(NUMERIC, "image") { description = "Input image, with shape [batch, height, width, channels]" }
+        Input(NUMERIC, "cropBoxes") { description = "Float32 crop, shape [numBoxes, 4] with values in range 0 to 1" }
+        Input(NUMERIC, "boxIndices") { description = "Indices: which image (index to dimension 0) the cropBoxes belong to. Rank 1, shape [numBoxes]" }
+        Input(NUMERIC, "cropOutSize") { description = "Output size for the images - int32, rank 1 with values [outHeight, outWidth]" }
+        Arg(NUMERIC, "extrapolationValue") { description = "Used for extrapolation, when applicable. 0.0 should be used for the default" }
+
+        Output(NUMERIC, "output"){ description = "Cropped and resized images" }
+
+        Doc(Language.ANY, DocScope.ALL){
+            """
+ Given an input image and some crop boxes, extract out the image subsets and resize them to the specified size.
+
+ @param image              Input image, with shape [batch, height, width, channels]
+ @param cropBoxes          Float32 crop, shape [numBoxes, 4] with values in range 0 to 1
+ @param boxIndices         Indices: which image (index to dimension 0) the cropBoxes belong to. Rank 1, shape [numBoxes]
+ @param cropOutSize        Output size for the images - int32, rank 1 with values [outHeight, outWidth]
+ @param extrapolationValue Used for extrapolation, when applicable. 0.0 should be used for the default
+ @return Cropped and resized images
+     
+""".trimIndent()
+        }
+    }
+
+    Op("extractImagePatches") {
+        javaPackage = "org.nd4j.linalg.api.ops.impl.image"
+        javaOpClass = "ExtractImagePatches"
+        Input(NUMERIC, "image") { description = "Input image to extract image patches from - shape [batch, height, width, channels]" }
+        Arg(INT, "kSizes") { count = AtLeast(0); description = "Kernel size - size of the image patches, [height, width]" }
+        Arg(INT, "strides") { count = AtLeast(0);description = "Stride in the input dimension for extracting image patches, [stride_height, stride_width]" }
+        Arg(INT, "rates") { count = AtLeast(0); description = "Usually [1,1]. Equivalent to dilation rate in dilated convolutions - how far apart the output pixels\n" +
+                "                 in the patches should be, in the input. A dilation of [a,b] means every {@code a}th pixel is taken\n" +
+                "                 along the height/rows dimension, and every {@code b}th pixel is take along the width/columns dimension" }
```

Comment:
Let's use `count = Exactly(2)` here

Note that the count types we can use are listed here:
https://github.com/KonduitAI/dl4j-dev-tools/blob/master/codegen/src/main/kotlin/org/nd4j/codegen/api/Variables.kt#L28-L33